### DISCUSSION TITLE
Config PSRAM

### DISCUSSION THREAD

**Discussion by workpage2** [espressif/arduino-esp32#7916] [Q&A]
How to configure CS and CLK pins for PSRAM?

**Comment by me-no-dev** [comment]
You can not. At least not in Arduino. You can look into ESP-IDF and using Arduino as component if you want such flexibility.

**Comment by workpage2** [comment]
OK. What pins are used in arduino? CS - GPIO16, CLK - GPIO17?

**Reply by me-no-dev** [reply]
https://www.espressif.com/sites/default/files/documentation/esp32-wrover-e_esp32-wrover-ie_datasheet_en.pdf you can find the schematics there :)
